packages/crayons-core/src/components/icon/library.system.ts

```diff
@@ -0,0 +1,91 @@
+import type { FwIconLibrary } from './library.icon.utils';
+//
+// System icons are a separate library to ensure they're always available, regardless of how the default icon library is
+// configured or if its icons resolve properly.
+//
+// All Crayons components must use the system library instead of the default library. For visual consistency, system
+// icons are a subset of Crayons Icons.
+//
+const crayons_system_icons = {
+  'add-contact':
```

Comment:
@detiwari003 Shall we restrict this list to have what is used within the crayons component and move out other icons like add-contact! 

The ones used inside the readme can point to crayons.

Reply:
yes.

Reply:
Changed to 10 files. These are those central components which are indispensable.

<img width="208" alt="Screenshot 2021-12-15 at 10 41 07 PM" src="https://user-images.githubusercontent.com/76547218/146232732-c80a6daf-2502-4d4f-8e3d-95bacbede1bf.png">

Reply:
Fixed